governance/executive-council.md

```diff
@@ -0,0 +1,264 @@
+This process ensures the Council has the skills and expertise necessary to meet
+the evolving pyOpenSci needs.
+
+### Executive Council selection process steps
+
+1. **Identify organization needs**:
+   - The Council first evaluates its current composition to identify areas where
+     additional expertise, perspectives, or skills could be useful to support the
+     organization’s mission and strategy.
+
+2. **Identify Candidates**:
+   - Potential candidates are identified based on their experience, expertise,
+     and alignment with pyOpenSci’s mission, vision, and values. This may include
+     leaders in the open science community, individuals with financial or
+     governance expertise, or others with relevant nonprofit governance skills.
+
+3. **Review and nominate the new member**:
+   - Candidates are discussed and nominated by current Council members during a
+     meeting or through the Executive Council Slack channel. After nomination, the Executive Director
+     will contact the candidate to see if they are interested/available.
+
+5. **Onboard the new candidate**:
+   - Once approved, new members are onboarded following the onboarding process
+     outlined in this document.
+
+### Terms and evaluation
+
+- Executive Council members are expected to serve for 2–3 years.
+- At the end of each 2-year term, members may choose to step down or renew their
+  commitment based on mutual agreement with the Council.
+
+As needed, the Executive Council may conduct periodic evaluations to ensure its composition continues
+to meet the organization’s needs.
+
+### Decision-making process
+
+The topics above require a motion and a vote for approval. Motions and votes can be made in Executive Council meetings or via email, Slack of GitHub. Motions and votes will be noted in the Executive Council meeting minutes.
+
+:::{admonition} Executive Council Approval
+:class: important
+
+The above governance for the Executive Council was last updated, voted on, and approved on **insert-date-here**. All content below this section can be modified as needed and does not require a vote.
+:::
+
+## Executive Council operations
+
+### Collaboration between the Executive Director and the Executive Council
+
+The Executive Director may request feedback
+from the Executive Council on significant or public-facing decisions; these decisions do not require a formal vote. These discussions generally
+occur in the **executive-council** Slack channel to ensure
+collaborative input. Examples include:
+
+- Significant public-facing changes to the pyOpenSci website.
+- Operational policies for staff (unless they carry significant financial implications).
+- Strategic decisions about pursuing proposals or writing grants.
+- Substantial changes to pyOpenSci programs.
+- Strategic decisions about pursuing proposals or writing grants.
+- Substantial changes to pyOpenSci programs.
```

Comment:
```suggestion
- Significant public-facing changes to the pyOpenSci website.
- Operational policies for staff (unless they carry significant financial implications).
- Strategic decisions about pursuing proposals or writing grants.
- Substantial changes to pyOpenSci programs.
```

Reply:
Good catch @kcranston that must have happened when i moved things around!! i totally missed it.